packages/lit-dev-content/site/docs/v3/components/properties.md

```diff
@@ -269,6 +271,17 @@ When converting a string-valued attribute into a property, Lit's default attribu
 
 When using TypeScript, this field should generally match the TypeScript type declared for the field. However, the `type` option is used by the Lit's _runtime_ for string serialization/deserialization, and should not be confused with a _type-checking_ mechanism.
 
+</dd>
+<dt id="use-default">
+
+`useDefault`
+
+</dt>
+<dd>
+
+Set to true to prevent initial attribute reflection for the default value when `reflect` is set to true, and to reset the property to its default value when its corresponding attribute is removed. For more information, see [Enabling attribute reflection](#reflected-attributes).
```

Comment:
Do we want a specific note here that the initial value is used as the default, and that it's retained in memory?

Reply:
Added